PyRDF/Proxy.py

```diff
@@ -136,5 +136,13 @@ def _create_new_op(self, *args, **kwargs):
         # Return the appropriate proxy object for the node
         if op.is_action():
             return ActionProxy(newNode)
-        else:
+        elif op.is_transformation():
             return TransformationProxy(newNode)
+        else:
+            try:
+                generator = CallableGenerator(self.proxied_node.get_head())
+                current_backend.execute(generator, trigger_loop=False)
+            except TypeError as e:
+                self.proxied_node.children.remove(newNode)
+                raise e
+            return newNode.ResultPtr
```

Comment:
In this case `newNode.ResultPtr` will not hold a `RResultPtr` but one of the types returned by the "other operations" of ROOT RDataFrame, such as strings or vectors. Only `Display` makes an exception, but it is not included in this PR.

Furthermore, while I get that creating a new node also for info operations can guarantee the execution of the same event loop only once via the `generator.execute` function, I don't think that these kind of operations are suited to be "nodes" in the graph, they are just querying the dataframe for metadata